src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String CHECK_LETTER = "w";
+    private static final String CHECK_REGEX = "[!\\.,?]";
+    private static final String SPACE_DIVIDER = " ";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder textFromFile = new StringBuilder();
+        FileReader fileReader;
+        try {
+            fileReader = new FileReader(fileName);
+            BufferedReader bufferedReader = new BufferedReader(fileReader);
+            String line = bufferedReader.readLine();
+            StringBuilder stringBuilder = new StringBuilder();
+            while (line != null) {
+                stringBuilder.append(line).append(SPACE_DIVIDER);
+                line = bufferedReader.readLine();
+            }
+            String[] words = stringBuilder.toString().split(SPACE_DIVIDER);
+            for (String string : words) {
+                if (string.toLowerCase().startsWith(CHECK_LETTER)) {
+                    textFromFile.append(string.toLowerCase()
+                            .replaceAll(CHECK_REGEX, "")).append(SPACE_DIVIDER);
+                }
+            }
```

Comment:
what about this comment? https://github.com/mate-academy/jv-read-from-file/pull/209#discussion_r607083274